Makefile

```diff
@@ -77,7 +81,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
-	$(CONTAINER_TOOL) build -t ${IMG} .
+	$(CONTAINER_TOOL) build -t ${IMG} . --platform linux/${TARGETARCH}
```

Comment:
Please use `GOARCH` here and remove `TARGETARCH`.

The default should always be to use whatever is in `go env`.

Developers can override that behaviour on an as-needed basis via `GOARCH=amd64 make docker-build`. It's just a development detail to be aware of.